qa-covid19.planx-pla.net/etlMapping.yaml

```diff
@@ -18,6 +18,11 @@ mappings:
       - name: reporting_org
       - name: state_hospital_region
       - name: state_region_description
+      - name: npi
+      - name: provider_first_line_business_pr
+      - name: provider_second_line_business_pr
```

Comment:
these 2 are not in the portal config

Reply:
https://docs.google.com/spreadsheets/d/1Cdpyi6hzdoKqLuvvB4bNGkJPS81pVzb-RchzajHLydE/edit#gid=235693014
it is not in the specification

Reply:
yeah but the spreadsheet is just a guide, there's no need to add to the ETL mapping if you don't use it 🤷

Reply:
for querying maybe? anyway, what are the titles to display?

Reply:
I found many fields in etlMapping but not in portal config anyway @paulineribeyre

Reply:
ideally only the fields are we show in the portal (as described in the spreadsheet) should be in the ETL mapping

Reply:
no one uses the query